vcl_templates/_multivariate_tests.vcl.erb

```diff
@@ -5,11 +5,11 @@
 # Begin dynamic section
 <% if ab_tests -%>
 if (req.http.Cookie ~ "cookies_policy") {
-  if (req.http.Cookie:cookies_policy ~ "%22usage%22:true") {
+  if (req.http.Cookie ~ "cookies_policy=%22usage%22:true") {
```

Comment:
Unless this means "get the `cookies_policy` cookie value and see if it has `%22usage%22:true` as a substring, it's not correct.

Reply:
Yep I would need to add for that 😉